MASTER3in1_v2025.08.05_v0.2.py

```
@@ -0,0 +1,1222 @@
+                pivot_df.columns = [str(h).strip().replace('↑','').strip() for h in pivot_df.columns]
+
+                pivot_df.rename(columns={
+                    'SuperOffice': 'Super_Office', 'Super Office': 'Super_Office',
+                    'TimeSheet: Owner Name': 'Owner_Name', 'Time Sheet: Owner Name': 'Owner_Name',
+                    'SumofSum of Hours': 'Sum_of_Hours', 'Sum of Sum of Hours': 'Sum_of_Hours'
+                }, inplace=True)
+
+                pivot_df.dropna(subset=['Owner_Name'], inplace=True)
+
+                self.non_reporters_data = {}
+                for index, row in pivot_df.iterrows():
```

Ruff (0.12.2)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)